package.json

```diff
@@ -15,7 +15,8 @@
         "typecheck": "concurrently --raw yarn:typecheck:*",
         "typecheck:app": "tsc --noEmit",
         "prepare": "husky install",
-        "build": "cd chain && npx tsx ../src/scripts/deployAndLinkLibs.js",
+        "deploy-libraries": "./scripts/deployLibraries.sh",
```

Comment:
@kentkolze I don't believe we have to deploy libraries anymore. Forge takes care of it using the `forge script`